validators/self-recharging-validator-wallets.adoc

```diff
@@ -0,0 +1,139 @@
+= Self-recharging Validator Wallets
+:doctype: article
+:revremark: preview
+:icons: font
+:toc: macro
+
+ifdef::env-github[]
+
+:tip-caption: :bulb:
+:note-caption: :information_source:
+:important-caption: :heavy_exclamation_mark:
+:caution-caption: :fire:
+:warning-caption: :warning:
+
+toc::[]
+
+endif::[]
+
+The SKALE Network uses a self-recharging wallet system to make it easier for validators and SKALE Chain Owners to maintain a minimum amount of ETH in each node wallet to fund node-related transactions.
+
+The Wallets contract in SKALE Manager manages this functionality.
+
+=== Types of wallets
+
+There are several types of wallets in the SKALE Network.
+
+Controlled by the validator, node operator, and SKALE Chain owner (these may be software, or hardware wallets):
```

Comment:
1- Controlled by the validator... 
2- Self-recharging wallet system.... 

It would be good to have the part "self-recharging wallets" with the numbering system visually shows that it's an addition to the types of wallets.